docs/ECLWatch/TheECLWatchMan.xml

```diff
@@ -251,10 +251,12 @@
             </mediaobject>
           </figure></para>
 
-        <para>You can search ECL Workunits, DFU Workunits, Logical Files, and
-        Queries using the global search box. The global search box also
-        supports using wildcards. To limit or filter your search results you
-        can use keywords as displayed in the empty search box.</para>
+        <para>You can search DFU Workunits, Logical Files, and Queries using
+        the global search box. The global search excludes ECL by default, you
+        can still search for ECL but must specify to do so. The global search
```

Comment:
I think that it still finds ECL Workunits (for example, search by Job Name, WUID, etc), but by default, the ECL inside that WU is not searched unless you specify to do so. However, you don't tell us how to specify that.